docs/source/tutorial.rst

```diff
@@ -1525,42 +1521,79 @@ construct key, by any of the following techniques:
    :caption: *Get the "latitude" metadata construct via its construct
              key and the 'get' method.*
 	     
-   >>> key = t.construct_key('latitude')
+   >>> key = t.construct('latitude', key=True)
    >>> c = t.constructs.get(key)
    <AuxiliaryCoordinate: latitude(10, 9) degrees_N>
 
-The `~Field.construct` method of the field construct and the
-`~Constructs.value` method of the `Constructs` instance will raise an
-exception of there is not a unique metadata construct to return, but
-this may be replaced with returning a default value or raising a
-customised exception:
+* with the `~Constructs.value` method of a `Constructs` instance, or
```

Comment:
Since when I look at the rendered docs this seems to be the final item in the list:

```suggestion
* with the `~Constructs.value` method of a `Constructs` instance.
```